src/super_gradients/training/models/detection_models/customizable_detector.py

```diff
@@ -182,18 +183,39 @@ def predict(
         conf: Optional[float] = None,
         batch_size: int = 32,
         fuse_model: bool = True,
+        target_bboxes: Optional[List[np.ndarray]] = None,
```

Comment:
should be `Optional[Union[np.ndarry, List[np.ndarray]]] = None`

Reply:
fixed 0798978413fab364fd7d89ea23baeab266c80e77